app/(app)/session/[session].tsx

```diff
@@ -1,40 +1,184 @@
+                  <Space size={5} horizontal />
+                  <StyledText size="sm" font="regular" style={{ color: colors.tertiary }}>
+                    {session?.speakers && session?.speakers.length > 1 ? 'Speakers' : 'Speaker'}
+                  </StyledText>
+                </View>
+                <StyledText size="lg" font="bold" style={{ color: dark ? colors.text : colors.primary }}>
+                  {session?.speakers.map((speaker) => speaker.name).join(', ')}
+                </StyledText>
+              </View>
+              <View>
+                <Space size={14} />
+                <FontAwesome5 name="star" size={24} color={colors.primary} />
+              </View>
+            </Row>
+
+            <Space size={20} />
+
+            <StyledText size="lg" font="bold">
+              {session?.title}
+            </StyledText>
+
+            <Space size={16} />
+
+            {session?.speakers && session?.speakers.length > 1 ? (
+              session?.speakers.map((speaker, index) => (
+                <View key={index.toString()}>
+                  <StyledText font="regular" style={{ color: colors.primary }}>
+                    {speaker.name} - {''}
+                    <StyledText font="regular" style={{ color: colors.textLight }}>
+                      {!showMoreBio ? truncate(140, speaker.biography) : speaker.biography}
+                      {speaker.biography && speaker.biography.length > 140 && (
+                        <StyledText
+                          size="sm"
+                          font="semiBold"
+                          style={{ color: colors.primary }}
+                          onPress={() => setShowMoreBio(!showMoreBio)}
+                        >
+                          {showMoreBio ? ' ...Show less' : ' Show more'}
+                        </StyledText>
+                      )}
+                    </StyledText>
+                  </StyledText>
+                  <Space size={10} />
+                </View>
+              ))
+            ) : (
+              <StyledText font="regular" style={{ color: colors.textLight }}>
+                {!showMoreBio ? truncate(140, session?.speakers[0]?.biography) : session?.speakers[0]?.biography}
+                {session?.speakers[0]?.biography && session?.speakers[0]?.biography.length > 140 && (
+                  <StyledText
+                    size="sm"
+                    font="semiBold"
+                    style={{ color: colors.primary }}
+                    onPress={() => setShowMoreBio(!showMoreBio)}
+                  >
+                    {showMoreBio ? ' ...Show less' : ' Show more'}
+                  </StyledText>
+                )}
+              </StyledText>
+            )}
+
+            <Space size={16} />
+
+            <Image
+              source={{ uri: session?.session_image || '' }}
+              style={styles.image}
+              contentFit="fill"
```

Comment:
The images looks squashed because of `contentFit="fill"`. We should give them the same treatment as we do on the home screen with `cover`.

<img width="573" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/e0ca60f3-bfd9-4418-bdfb-8153ba80e3c0">

Reply:
I converted to `cover` with `contentPosition="left"` like on home screen, but because of width constraints, this may have uneven edges. if I remove `contentPosition="left"` the text in the image is cut. then again since the title is already in the page should that be an issue. please advice